### PR TITLE
Use new EventDatabaseProvider implementation

### PR DESCRIPTION
**Pull request by dannykruitbosch**
The getting started local.conf was not yet updated to use the new EventDatabaseProvider implementation. This prevented the engine to start in the getting started.

### CODE REVIEW

File: src/conf/local.conf [hunk 1]
```diff
@@ -191,7 +191,7 @@ cafienne {
 ##                                                                                   ##
 #######################################################################################
 akka-persistence-jdbc {
```
Comment (tpetter) [inline src/conf/local.conf:193]: Kindly first rebase on main ;)

**Review by tpetter — CHANGES_REQUESTED** [review]

File: src/conf/local.conf (hunk 1 above)
Comment (dannykruitbosch) [reply src/conf/local.conf:193]: Done

**Comment by dannykruitbosch** [comment]
@tpetter seems my PR is not needed anymore since #24 contains the same fix. Close this one if you agree.

**Comment by tpetter** [comment]
> @tpetter seems my PR is not needed anymore since #24 contains the same fix. Close this one if you agree.

Agreed

**Pull request closed without merging**